packages/plugins/Gfi/src/components/List.vue

```diff
@@ -0,0 +1,159 @@
+<!-- eslint-disable vue/no-v-html -->
+<template>
+  <v-scroll-x-reverse-transition>
+    <v-card class="plugin-gfi-list">
+      <v-card-title>
+        <v-icon>fa-list</v-icon>
+        {{ $t('common:plugins.gfi.list.header') }}
+      </v-card-title>
+      <v-card-subtitle v-if="maxPage > 1">
+        {{
+          `${$t('common:plugins.gfi.list.entry')} ${page * pageLength + 1} ${$t(
+            'common:plugins.gfi.list.to'
+          )} ${Math.min((page + 1) * pageLength, listFeatures.length)} ${$t(
+            'common:plugins.gfi.list.of'
+          )} ${listFeatures.length}`
+        }}
+        <v-pagination
+          v-model="_page"
+          color="primary"
```

Comment:
![disabled](https://github.com/Dataport/polar/assets/73653210/c4d49f5c-4f1e-4ef8-99aa-a366c4cb28ee)

The difference between the disabled and active forward / backward button is almost non-existand.

Reply:
```
<style lang="scss">
#polar-plugin-gfi .v-pagination button {
  transition: opacity 200ms;

  &:disabled {
    opacity: 0.3;
  }
}
</style>
```

This in List.vue makes things a little more extreme. But I don't consider it particularly helpful, since the almost-nonexistance may be confusing, too. Setting visibility to hidden to altogether remove it without jumps doesn't seem quite right, either.